blockchain/blockchain_test.go

```diff
@@ -652,7 +652,7 @@ func TestRevert(t *testing.T) {
 	require.NoError(t, chain.RevertHead())
 	require.NoError(t, chain.RevertHead())
 
-	t.Run("empty blockchain should mean empty db", func(t *testing.T) {
+	t.Run("empty blockchain should mean db with p2p hash only", func(t *testing.T) {
```

Comment:
Since p2p block hashes are no longer stored in the DB, I don't think this change is required, right?